### ISSUE TITLE
Portable VST3 Windows broken now we moved to a bundle

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge#5825]
But I'm fixing this. Just tagging an issue in.

**Comment by baconpaul** [comment]
@mkruselj I know you've been poking about for issues to fix and I think this one is actually pretty easy but requires some windows testing

Right now in SurgeStorage we do code like this

```
 if (auto path{installPath / L"SurgeXTData"}; fs::is_directory(path))
        {
            datapath = std::move(path);
        }
```

The problem is installPath is now the DLL path. So that's too low. Two choices

1. We know we are in a bundle so search `installPath.parent_path().parent_path() / L"SurgeXTData"`. Pro - simple. Con - if people unpack the bundle for old daws it breaks
2. We could recurse up looking. I like this better. So

```cpp
auto cp{installPath};
while( datapath.empty() && cp.has_parent_path() && cp != cp.parent_path())
{
    auto portable = cp / L"SurgeXTData";
    // probably want to print portable here
    if (fs::is_directory(portable))
    {
       datapath = std::move(portable);
    }
    cp = cp.parent_path()
}
```

basically climb up the filesystem looking. You can do the same with the user path also just with different names.

Your option. I can also do it when I'm next in windows. In that case these notes will help me then too.

**Comment by mkruselj** [comment]
I will give it a shot, thanks!
